model/qti/templates/qti.element.tpl.php

```diff
@@ -1,28 +1,24 @@
 <?php
-/*
+/*  
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
  * as published by the Free Software Foundation; under version 2
  * of the License (non-upgradable).
- *
+ * 
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
- *
+ * 
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- *
+ * 
  * Copyright (c) 2013 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
- *
- *
+ *               
+ * 
  */
 $attributes = get_data('attributes');
-$attributes = empty($attributes) ? '' : ' ' . $attributes;//add space if not empty
+$attributes = empty($attributes) ? '' : ' '.$attributes;//add space if not empty
 ?>
-<?php if (trim(get_data('body') == '')) :
-    ?><<?=get_data('tag')?><?=$attributes?>/><?php
-else :
-    ?><<?=get_data('tag')?><?=$attributes?>><?=get_data('body')?></<?=get_data('tag')?>><?php
-endif;?>
+<?php if(trim(get_data('body') == '')):?><<?=get_data('tag')?><?=$attributes?>/><?php else:?><<?=get_data('tag')?><?=$attributes?>><?=get_data('body')?></<?=get_data('tag')?>><?php endif;?>
```

Comment:
Why don't you split this line?

Reply:
I have reverted it as before, as it is template, i will not take time to apply cs-fixer